src/tests/Interop/CMakeLists.txt

```diff
@@ -102,3 +102,5 @@ if(CLR_CMAKE_TARGET_OSX OR CLR_CMAKE_TARGET_IOS OR CLR_CMAKE_TARGET_TVOS)
     add_subdirectory(ObjectiveC/AutoReleaseTest)
     add_subdirectory(ObjectiveC/ObjectiveCMarshalAPI)
 endif()
+
+add_subdirectory(MonoAPI/Native)
```

Comment:
Let's move this up to before the first `if` branch since it should be always done and with that "group".